src/deploy.ts

```diff
@@ -153,24 +166,34 @@ async function runDeploy(
 
   newDeploy.bytecodeHash = crypto
     .createHash("sha256")
-    .update(
-      JSON.stringify(
-        JSON.parse(
-          fs.readFileSync(`out/${contract}.sol/${contract}.json`, "utf-8"),
-        ).bytecode.object,
-      ),
-    )
+    .update(JSON.stringify(contractJson.bytecode.object))
     .digest("hex");
   newDeploy.abiHash = crypto
     .createHash("sha256")
-    .update(
-      JSON.stringify(
-        JSON.parse(
-          fs.readFileSync(`out/${contract}.sol/${contract}.json`, "utf-8"),
-        ).abi,
-      ),
-    )
+    .update(JSON.stringify(contractJson.abi))
     .digest("hex");
+  newDeploy.commitHash = getLatestCommitHash();
+
+  if (storeAbi) {
+    // Store ABI file and generate diff
+    const abiDir = `deployments/abi/${contract}`;
+    if (!fs.existsSync(abiDir)) {
+      fs.mkdirSync(abiDir, { recursive: true });
+    }
+    const newAbiPath = `${abiDir}/v${newDeploy.version.replace(/\./g, "_")}.json`;
```

Comment:
How does this fit into minor version changes?

Reply:
Ah good catch